packages/eslint-config-terra/package.json

```
@@ -0,0 +1,49 @@
+{
+  "name": "eslint-config-terra",
+  "version": "3.10.0",
```

Are we scoping the packages?

```suggestion
  "name": "@cerner/eslint-config-terra",
  "version": "4.0.0",
```

---

I still think we should scope it in this pr though

---

https://github.com/cerner/terra-toolkit/pull/402/commits/81534d2ce1c15eac348e43573e0b42148d28f719